### PR TITLE
feat(blog): Theme Jam wrap-up post

### PR DESCRIPTION
**Pull request by jlengstorf**
- talk about how we reviewed submissions
- announce the community members who will get all-expenses-paid trips to Gatsby Days
- celebrate all the amazing people in the community who participated

### CODE REVIEW

File: docs/blog/2019-08-07-theme-jam/index.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+title: "Gatsby Theme Jam: Congratulations Vojtěch Holík and Allan Pooley!"
+date: 2019-08-07T12:00:00
+author: Jason Lengstorf
+excerpt: >
+  During the Theme Jam, 111 themes wer created by our incredible community. Join us in celebrating the Gatsby community (that’s you!) and congratulating Vojtěch and Allan!
```
Comment (johno) [inline docs/blog/2019-08-07-theme-jam/index.md]:
```suggestion
  During the Theme Jam, 111 themes were created by our incredible community. Join us in celebrating the Gatsby community (that’s you!) and congratulating Vojtěch and Allan!
```

File: docs/blog/2019-08-07-theme-jam/index.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+tags:
+  - themes
+  - community
+---
+
+At the beginning of July, [Brent Jackson](https://twitter.com/jxnblk) joined _Learn With Jason_ to show [how to use `theme-ui`](https://youtu.be/6Z4p-qjnKCQ?list=PLz8Iz-Fnk_eTpvd49Sa77NiF8Uqq5Iykx). During the show, we announced the stable launch of [Gatsby themes](/docs/themes/) and kicked off the [Theme Jam](https://themejam.gatsbyjs.org), a celebration of our incredible community of creators.
+
+## What is the Theme Jam?
+
+In a nutshell, the Theme Jam was a way for Gatsby to give back to its community: everyone who published a theme and submitted it to the Theme Jam will receive exclusive swag designed by [Maggie Appleton](https://maggieappleton.com/), and two creators who published themes our team reviewed as exemplary are getting an all-expenses paid trip to [Gatsby Days](https://www.gatsbyjs.com/resources/gatsby-days/). (For more details, see the [rules](https://themejam.gatsbyjs.org/rules).)
```
Comment (johno) [inline docs/blog/2019-08-07-theme-jam/index.md]: Maybe we could split this into two sentences?

Comment (jlengstorf) [reply docs/blog/2019-08-07-theme-jam/index.md]: Good call. Done!

File: docs/blog/2019-08-07-theme-jam/index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+![Theme Jam illustration by Maggie Appleton.](images/theme-jam.png)
+
+In a nutshell, the Theme Jam was a way for Gatsby to give back to its community: everyone who published a theme and submitted it to the Theme Jam will receive exclusive swag designed by [Maggie Appleton](https://maggieappleton.com/). Two creators who published themes our team reviewed as exemplary are getting an all-expenses paid trip to [Gatsby Days](https://www.gatsbyjs.com/resources/gatsby-days/). (For more details, see the [rules](https://themejam.gatsbyjs.org/rules).)
+
+## How were entries reviewed?
+
+Every Theme Jam entry needed to have a demo site, a published npm package, a public repo, and a properly configured `package.json`.
+
+Entries also needed to be incredibly performant (something something blazing fast) and meet accessibility standards.
```
Comment (amberleyromo) [inline docs/blog/2019-08-07-theme-jam/index.md]:
```suggestion
Entries also needed to be incredibly performant (_something something blazing fast_) and meet accessibility standards.
```

File: docs/blog/2019-08-07-theme-jam/index.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+On top of that, submissions were code reviewed for documentation, the ability to extend and shadow the themes, and creativity.
+
+### Part 1: automated review
```
Comment (amberleyromo) [inline docs/blog/2019-08-07-theme-jam/index.md]:
```suggestion
### Part 1: Automated Review
```

Is there a reason these are lower-cased?

File: docs/blog/2019-08-07-theme-jam/index.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+To pass the automated review, a theme must:
+
+- score at least 90 on the Lighthouse accessibility audit
+- score at least 80 on the Lighthouse performance audit
+- have all required fields in `package.json`
+
+Of the 111 submissions, 52 passed the automated review.
+
+### Part 2: manual review
```
Comment (amberleyromo) [inline docs/blog/2019-08-07-theme-jam/index.md]:
```suggestion
### Part 2: Manual Review
```

File: docs/blog/2019-08-07-theme-jam/index.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+Every theme that cleared the automated review was manually reviewed by [Jackson](https://twitter.com/jxnblk), [Amberley](https://twitter.com/amber1ey), [John](https://twitter.com/4lpine), [Kyle Gill](https://gill_kyle), [Chris](https://twitter.com/chrisbiscardi), [Flo](https://twitter.com/fk), and me.
```
Comment (amberleyromo) [inline docs/blog/2019-08-07-theme-jam/index.md]:
```suggestion
Every theme that cleared the automated review was manually reviewed by [Jackson](https://twitter.com/jxnblk), [Amberley](https://twitter.com/amber1ey), [John](https://twitter.com/4lpine), [Kyle Gill](https://gill_kyle), [Chris](https://twitter.com/chrisbiscardi), [Flo](https://twitter.com/fk), and [me](https://twitter.com/jlengstorf).
```

File: docs/blog/2019-08-07-theme-jam/index.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+Amberley did a manual accessibility audit. All of the themes we reviewed has a few accessibility issues, so we will be opening issues with feedback and suggestions to improve that.
```
Comment (amberleyromo) [inline docs/blog/2019-08-07-theme-jam/index.md]:
```suggestion
Amberley did a manual accessibility audit. Nearly all of the themes we reviewed have a few accessibility issues, so we will be opening issues with feedback and suggestions to improve that.
```

File: docs/blog/2019-08-07-theme-jam/index.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+## Highlighting Exemplary Creators
+
+We plan to feature a lot of the excellent creators and their themes that were created both as part of Theme Jam and outside of it.
+
+For Theme Jam, we specifically wanted to call out excellence. This is really important with themes being so early; great examples lead to improving the quality of the entire ecosystem.
+
+Two of the themes we received stood out especially. As we reviewed, we all took notes on the themes we thought were outstanding, and _every one of us_ chose these two themes as excellent examples.
+
+**So please join us in congratulating Vojtěch Holík and Allan Pooley, who will receive an all-expenses paid trip to Gatsby Days in recognition of their outstanding work on [`gatsby-theme-simplecast`](https://github.com/vojtaholik/gatsby-theme-simplecast) and [`gatsby-theme-legals-prismic`](https://github.com/littleplusbig/gatsby-theme-legals-prismic)!**
```
Comment (amberleyromo) [inline docs/blog/2019-08-07-theme-jam/index.md]:
```suggestion
**So please join us in congratulating Vojtěch Holík and Allan Pooley on their outstanding work on [`gatsby-theme-simplecast`](https://github.com/vojtaholik/gatsby-theme-simplecast) and [`gatsby-theme-legals-prismic`](https://github.com/littleplusbig/gatsby-theme-legals-prismic)!**
```

Comment (amberleyromo) [reply docs/blog/2019-08-07-theme-jam/index.md]: I (personally) don't really see a need to highlight Gatsby Days again, the focus is on their work.

File: docs/blog/2019-08-07-theme-jam/index.md (hunk 4 above)
Comment (jlengstorf) [reply docs/blog/2019-08-07-theme-jam/index.md]: Force of habit. I didn't check the style guide. My own style is to title case H2s, but sentence case H3 and below.

**Comment by jlengstorf** [comment]
@m-allanson @DSchau do we need something special for this last CI check? Can y'all merge if this is good to go?

**Comment by DSchau** [comment]
I have been summoned. Will merge, and I'll fix that check to only be on master.

**Pull request merged**